docs/developers/deployment_guides/blockchain/oracles.md

```
@@ -35,7 +35,7 @@ You will need:
 
 - [OpenSSL](https://www.openssl.org/) - you will need OpenSSL installed to generate solution signing key. Linux: by default, Ubuntu: `apt install openssl`, MacOs: `brew install openssl`.
 
-- [SPCTL](/developers/cli_guides/) - our CLI tool, must be fully [configured](/developers/cli_guides/configure), including access to decentralized storage: it will be used to store encrypted configurations for the oracle script.
+- [SPCTL](/developers/cli_guides/) - our CLI tool, must be fully [configured](/cli), including access to decentralized storage: it will be used to store encrypted configurations for the oracle script.
```

Markdownlint (0.37.0)

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)